package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lambda-essentials-ts",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
It's kiiiinda breaking. I can be convinced this is 2.0.0 as well. I only bumped minor here as the change is in an optional field of type any. So TS wouldn't let you write code that would break with this change

Reply:
I would make it 2.0.0 if it requires any code changes in code integrating this library. Normally we won't have this knowledge, but I know about a place that relies on the wrapping `error` property.

Also, don't forget to update the changelog file :)

Reply:
No objection & thanks for the reminder!